libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat

```diff
@@ -0,0 +1,343 @@
+
+# USB
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+PA9 VBUS INPUT OPENDRAIN
+
+# pins for SWD debugging
+PA13 JTMS-SWDIO SWD
+PA14 JTCK-SWCLK SWD
+
+# telem1
+PE8 UART7_TX UART7
+PF6 UART7_RX UART7
+PF8 UART7_RTS UART7
+PE10 UART7_CTS UART7
+
+# telem2
+PC8 UART5_RTS UART5
+PC9 UART5_CTS UART5
+PC12 UART5_TX UART5
+PD2 UART5_RX UART5
+
+# telem3
+PA3 USART2_RX USART2
+PD5 USART2_TX USART2
+PD3 USART2_CTS USART2
+PD4 USART2_RTS USART2
+#define DEFAULT_SERIAL5_PROTOCOL SerialProtocol_MAVLink2
+
+# GPS1
+PB6 USART1_TX USART1
+PB7 USART1_RX USART1
+
+# GPS2
+PE0 UART8_RX UART8
+PE1 UART8_TX UART8
+
+# uart4
+PH13 UART4_TX UART4
+PH14 UART4_RX UART4
+
+# debug uart
+PD8 USART3_TX USART3
+PD9 USART3_RX USART3
+
+# USART6 is for IOMCU
+PC6 USART6_TX USART6
+PC7 USART6_RX USART6
+IOMCU_UART USART6
+
+# Ethernet
+PC1 ETH_MDC              ETH1
+PA2 ETH_MDIO             ETH1
+PC4 ETH_RMII_RXD0        ETH1
+PC5 ETH_RMII_RXD1        ETH1
+PG13 ETH_RMII_TXD0       ETH1
+PG12 ETH_RMII_TXD1       ETH1
+PB11 ETH_RMII_TX_EN      ETH1
+PA7 ETH_RMII_CRS_DV      ETH1
+PA1 ETH_RMII_REF_CLK     ETH1
+#PG15 ETH_POWER_EN       ETH1
+
+PG15 Ethernet_PWR_EN OUTPUT HIGH # disable power on ethernet
+
+define BOARD_PHY_ID  MII_LAN8742A_ID
+define BOARD_PHY_RMII
+
+# ADC
+PA0 SCALED1_V3V3 ADC1 SCALE(2)
+PA4 SCALED2_V3V3 ADC1 SCALE(2)
+PB0 SCALED3_V3V3 ADC1 SCALE(2)
+PF12 SCALED4_V3V3 ADC1 SCALE(2)
+
+PB1 VDD_5V_SENS  ADC1 SCALE(2)
+
+# pin7 on AD&IO, analog 12
+PC2 ADC1_6V6 ADC1 SCALE(2)
+
+# pin6 on AD&IO, analog 13
+PC3 ADC1_3V3 ADC1 SCALE(1)
+
+# SPI1 - IMU3 ICM45686
+PA5 SPI1_SCK SPI1
+PB5 SPI1_MOSI SPI1
+PG9 SPI1_MISO SPI1
+PI9 SP1_CS1 CS
+
+# SPI2 -IMU1 ICM45686
+PI1 SPI2_SCK SPI2
+PI2 SPI2_MISO SPI2
+PI3 SPI2_MOSI SPI2
+PH5 SP2_CS1 CS
+PA10 SP2_DRDY2 INPUT
+
+# SPI3 -IMU2 BMI088
+PB2  SPI3_MOSI SPI3
+PC10 SPI3_SCK  SPI3
+PC11 SPI3_MISO SPI3
+PI4 SP3_CS1 CS
+PI8 SP3_CS2 CS
+PI6 SP3_DRDY1 INPUT
+PI7 SP3_DRDY2 INPUT GPIO(93)
+define SP3_DRDY2 93
+
+# SPI4 - unused
+#PE12 SPI4_SCK SPI4
+#PE13 SPI4_MISO SPI4
+#PE14 SPI4_MOSI SPI4
+#PF3  SP4_DRDY1 INPUT
+PH15 SP4_CS1 CS
+
+# SPI5 - FRAM
+PF7  SPI5_SCK  SPI5
+PH7  SPI5_MISO SPI5
+PF11 SPI5_MOSI SPI5
+PG7  FRAM_CS CS
+
+# SPI6 - external1 (disabled to save DMA channels)
+# PB3  SPI6_SCK SPI6
+# PA6  SPI6_MISO SPI6
+# PG14 SPI6_MOSI SPI6
+# PI10   EXT1_CS CS
+
+# PWM output pins
+PI0  TIM5_CH4 TIM5 PWM(1) GPIO(50)
+PH12 TIM5_CH3 TIM5 PWM(2) GPIO(51)
+PH11 TIM5_CH2 TIM5 PWM(3) GPIO(52)
+PH10 TIM5_CH1 TIM5 PWM(4) GPIO(53)
+PD13 TIM4_CH2 TIM4 PWM(5) GPIO(54)
+PD14 TIM4_CH3 TIM4 PWM(6) GPIO(55) 
+
+# Pin for PWM Voltage Selection, 0 means 3.3v, 1 means 5v
+PG6 PWM_VOLT_SEL OUTPUT HIGH GPIO(3)
+define HAL_GPIO_PWM_VOLT_PIN 3
+define HAL_GPIO_PWM_VOLT_3v3 0
+
+# we need to disable DMA on the last 2 FMU channels
+# as timer 12 doesn't have a TIMn_UP DMA option
+PH6  TIM12_CH1 TIM12 PWM(7) GPIO(56) NODMA
+PH9  TIM12_CH2 TIM12 PWM(8) GPIO(57) NODMA
+
+# GPIOs
+PE11 FMU_CAP1 INPUT GPIO(58)

```

Comment:
which connector/pin name?

Reply:
![PH9 PE11](https://github.com/user-attachments/assets/d6fc96e0-27ea-4b10-b951-8fcbb94f704a)
PE11 is the CAP1 label on the AD&IO terminal, PH9 is the M16 label on the M15M16 terminal, which I have marked in this figure.